src/screens/FeedScreen/hooks/useCoreFeed.js

```diff
@@ -35,6 +30,16 @@ const useCoreFeed = () => {
   const {myProfile} = profileContext;
   const {bottom} = useSafeAreaInsets();
 
+  const {sendViewPostTimeWithFeeds, updateViewPostTime, isSamePostViewed} = useViewPostTimeHook(
+    dispatch,
+    timer,
+    viewPostTimeIndex
+  );
+
+  const {fetchNextFeeds} = useFeedPreloadHook(feeds.length, () => {
+    getDataFeeds(postOffset, false, nextTargetFeed);
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useViewPostTimeHook` and `useFeedPreloadHook` are being used correctly. However, it's important to ensure that the `dispatch`, `timer`, `viewPostTimeIndex`, and `feeds.length` passed as arguments to these hooks are always up-to-date and in sync with the component's lifecycle. If not, it could lead to unexpected behavior.



<!-- This is an auto-generated comment by OSS CodeRabbit -->